code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/templates/ManualCarouselPushTemplate.kt

```diff
@@ -17,7 +17,7 @@ import com.adobe.marketing.mobile.notificationbuilder.internal.util.Notification
 
 internal class ManualCarouselPushTemplate(data: NotificationData) : CarouselPushTemplate(data) {
     internal var intentAction: String? = null
-        private set
+        internal set
```

Comment:
Can we avoid making this `internal` so that it's not accidentally modified by the builder class? You can pass in whatever value you need in `data` , would that not work?

Reply:
reverted @spoorthipujariadobe